lightkurve/periodogram.py

```diff
@@ -566,23 +570,56 @@ def from_lightcurve(lc, minimum_frequency=None, maximum_frequency=None,
+        > 1) is useful for displaying the full dynamics of the spectrum, where
+        the frequencies and amplitudes can be measured directly from the plot
+        itself, with no fitting required. On the other hand, an
+        oversample_factor of 1 means the spectrum is critically sampled, where
+        every point in the spectrum is independent of the others. This is
+        useful for stars whose mode lifetimes are shorter than the time-base of
+        the data, which is a behaviour usually observed in solar-like
+        oscillators where Lorentzians are fit to modes in the power spectrum.
+        An oversample_factor of 1 is less likely to cause issues for these
+        stars, because the modes are usually fully resolved. That is, the power
+        from each mode is spread over a range of frequencies due to damping.
+        Hence, any small error from measuring mode frequencies by taking the
+        maximum of the peak is negligible compared with the intrinsic linewidth
+        of the modes.
+
+        The `normalization` parameter will normalize the spectrum to either
+        power spectral density ("psd") or amplitude ("amplitude"). Users doing
+        asteroseismology on stars that do not exhibit solar-like oscillations
+        (e.g. delta Scutis) typically prefer `normalization="amplitude"`
+        because "amplitude" has higher dynamic range (high and low peaks
+        visible simultaneously), and we often want to read off amplitudes from
+        the plot. If `normalization="amplitude"`, the default value for
+        `oversample_factor` is set to 5 and `freq_unit` is 1/day.
+        Alternatively, users doing asteroseismology on solar-like oscillators
+        tend to prefer `normalization="psd"` because power density has a scaled
+        axis that depends on the length of the observing time, and is used when
+        we are interested in noise levels (e.g. granulation) and to look at
+        damped oscillations. If `normalization="psd"`, the default value for
+        `oversample_factor` is set to 1 and `freq_unit` is set to microHz.
+        Default values of `freq_unit` and `oversample_factor` can be
+        overridden. See Appendix A of Kjeldsen & Bedding, 1995 for a full
+        discussion of normalization and measurement of oscillation amplitudes
+        (http://adsabs.harvard.edu/abs/1995A%26A...293...87K).
+
+        The parameter nterms controls how many Fourier terms are used in the
+        model. Setting the Nyquist_factor to be greater than 1 will sample the
+        space beyond the Nyquist frequency, which may introduce aliasing.
+
+        The `freq_unit` parameter allows a request for alternative units in frequency
+        space. By default frequency is in (1/day) and power in (amplitude
+        (ppm)). Asteroseismologists for example may want frequency in (microHz)
+        in which case they would pass `freq_unit=u.microhertz`.
```

Comment:
I love the detail captured in this docstring now.  :heart:  In the future we should think about walking the users through this information as part of a tutorial.

Reply:
Geert @barentsen I am happy to read through the doc but I am new to github and can't figure out how to see it!
Can you please help (or email tim,bedding@sydney.edu.au)?

Reply:
@timbedding Sure thing!  You can read the newly-proposed docstring at this url: https://github.com/KeplerGO/lightkurve/blob/466548846e0b818bd70738f48c52d5f40adbbd6b/lightkurve/periodogram.py#L561

Reply:
Oops, GitHub renders the url into a fancy widget.  The full url is: `https://github.com/KeplerGO/lightkurve/blob/466548846e0b818bd70738f48c52d5f40adbbd6b/lightkurve/periodogram.py#L561-L682`

Reply:
I've added some of Tim's suggestions to the docstring